### PR TITLE
Add Request.signal

### PR DESCRIPTION
**Pull request by teoli2003**
This is part of openwebdocs/project#152

I added:
- a new `Request.signal`
- an entry in the list of properties on `Request`.

The example is not live as it would need a server.

### CODE REVIEW

File: files/en-us/web/api/request/signal/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Request.signal
+slug: Web/API/Request/signal
+page-type: web-api-instance-property
+browser-compat: api.Request.signal
+---
+
+{{APIRef("Fetch API")}}
+
+The read-only **`signal`** property of the {{DOMxRef("Request")}} interface return the {{domxref("AbortSignal")}} associated with the request.
```
Comment (dawei-wang) [inline files/en-us/web/api/request/signal/index.md:10]:
```suggestion
The read-only **`signal`** property of the {{DOMxRef("Request")}} interface returns the {{domxref("AbortSignal")}} associated with the request.
```

File: files/en-us/web/api/request/signal/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+An {{DOMxRef("AbortSignal")}} object.
+
+## Examples
+
+```js
+// Create a new abort controller
+const controller = new AbortController();
+
+// Creatae a request we this controller's AbortSignal object
```
Comment (dawei-wang) [inline files/en-us/web/api/request/signal/index.md:22]:
```suggestion
// Create a request with this controller's AbortSignal object
```

**Review by dawei-wang — COMMENTED** [review]
A couple of typos

**Review by wbamberg — CHANGES_REQUESTED** [review]
Looks great except for the comments already made by @dawei-wang :)

(let me know of you would prefer me to just push the suggestions and merge the PR, in cases like this.)

File: files/en-us/web/api/request/signal/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const req = new Request('/', { signal: controller.signal });
+
+// Add an event handler logging a message in case of abort
+req.signal.addEventListener("signal",  () => { console.log('abort'); }
```
Comment (Josh-Cena) [inline files/en-us/web/api/request/signal/index.md:26]:
```suggestion
req.signal.addEventListener("signal", () => {
  console.log("abort");
});
```

**Comment by teoli2003** [comment]
> (let me know of you would prefer me to just push the suggestions and merge the PR, in cases like this.)

I like to see the grammar errors (or typos) so I can learn from them. But no worry either way.

**Review by wbamberg — APPROVED** [review]

**Pull request merged**